NASA-PDS/supplementer
# Import DOI data from Sqlite database

## 🗒️ Summary
Import DOI data from Sqlite database

## ⚙️ Test Data and/or Report

- Run following command to import data from Sqlite DOI database into the Registry

```
supplementer add-doi -file /tmp/my_doi.db
```

You can also pass optional parameters to configure Registry URL and authentication.
```
Usage: supplementer add-doi <options>

Add DOIs to already registered products.

Required parameters:
  -file <path>     Path to Sqlite database

Optional parameters:
  -auth <file>     Registry authentication configuration file
  -es <url>        Registry (OpenSearch) URL. Default is http://localhost:9200
  -index <name>    Registry index name. Default is 'registry'
```

- Check `ops:Identifiers/ops:doi` field in OpenSearch / Elasticsearch


## ♻️ Related Issues
https://github.com/NASA-PDS/doi-service/issues/249
